backend.native/tests/interop/objc/illegal_sharing_with_weak/objclib.m

```diff
@@ -0,0 +1,11 @@
+#import "objclib.h"
+
+static NSObject* __weak globalObject = nil;
+
+void setObject(NSObject* obj) {
+  globalObject = obj;
+}
+
+bool isObjectAliveShouldCrash() {
+  return globalObject != nil;
+}
```

Comment:
It might be enough to put these declarations to `.h`, making building `.dylib` unnecessary.

Reply:
That simplified things indeed. Thanks!